src/sdltiles.cpp

```diff
@@ -1246,6 +1252,49 @@ long sdl_keysym_to_curses( SDL_Keysym keysym )
     }
 }
 
+bool handle_resize(int w, int h)
+{
+    if( ( w != WindowWidth ) || ( h != WindowHeight ) ) {
+        WindowWidth = w;
+        WindowHeight = h;
+        TERMINAL_WIDTH = WindowWidth / fontwidth;
+        TERMINAL_HEIGHT = WindowHeight / fontheight;
+        SetupRenderTarget();
+        g->init_ui();
+        tilecontext->reinit_minimap();
+        
+        return true;
+    }
+    return false;
+}
+
+void toggle_fullscreen_window()
+{
+    static int restore_win_w = get_option<int>( "TERMINAL_X" ) * fontwidth;
+    static int restore_win_h = get_option<int>( "TERMINAL_Y" ) * fontheight;
+
+    if ( fullscreen ) {
+        if( SDL_SetWindowFullscreen( ::window.get(), 0 ) != 0 ) {
+            dbg(D_ERROR) << "SDL_SetWinodwFullscreen failed: " << SDL_GetError();
+            return;
+        }
+        SDL_RestoreWindow( ::window.get() );
+        SDL_SetWindowSize( window.get(), restore_win_w, restore_win_h );
```

Comment:
Why are you accessing the global `window` in an inconsistent way (with and without `::` prefix)? The prefix is not needed in this function all.

Reply:
I honestly have no idea. I was jumping between this function and WinCreate to work things out, I guess I got my wires crossed. Why exactly is the prefix being used in WinCreate?